code-challenge/src/components/Article/Article.lazy.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { lazy, Suspense } from 'react';
+import { ArticleListProps } from './Article';
+
+const LazyArticle = lazy(() => import('./Article'));
+
+const Article = (props: JSX.IntrinsicAttributes & ArticleListProps & { children?: React.ReactNode; }) => (
+  <Suspense fallback={null}>
+    <LazyArticle {...props} />
+  </Suspense>
+);
+
+export default Article;
```

Comment:
Love to see lazy loading! What would be a good scenario where I would use the Article directly instead of using the LazyArticle?

Reply:
If there were some pagination, I might use the first set of articles directly rather than using the lazy loader